vars/kibanaPipeline.groovy

```diff
@@ -181,7 +181,7 @@ def withGcsArtifactUpload(workerName, closure) {
   def ARTIFACT_PATTERNS = [
     'target/junit/**/*',
     'target/kibana-*',
-    'target/kibana-coverage/**/*',
+    'target/kibana-coverage/jest/**/*',
```

Comment:
We don't want to upload content of `target/kibana-coverage/functional` since it is 30+ GB and we archive it here 
https://github.com/elastic/kibana/blob/2494b245d3aa94144b67a71794b18f3511c2eb8b/vars/kibanaPipeline.groovy#L219-L223